js/plot/static/StaticPlot.svelte

```diff
@@ -37,6 +37,7 @@
 	{container}
 	{scale}
 	{min_width}
+	allow_overflow={false}
```

Comment:
I get the same behavior without this line?

Reply:
See bokeh_plot demo, the plot leaks out of the box